homeassistant/components/ovo_energy/strings.json

```
@@ -0,0 +1,19 @@
+{
+  "config": {
+    "error": {
+      "authorization_error": "Authorization error. Check your credentials.",
+      "connection_error": "Could not connect to OVO Energy."
+    },
+    "step": {
+      "user": {
+        "data": {
+          "username": "[%key:common::config_flow::data::username%]",
+          "password": "[%key:common::config_flow::data::password%]"
+        },
+        "description": "Set up an OVO Energy instance to access your energy usage.",
+        "title": "Add OVO Energy"
+      }
+    }
+  },
+  "title": "OVO Energy"
```

The name needs no translation (as it is a company/brand). It is fetched from the manifest.

```suggestion
  }
```